src/iuse.cpp

```diff
@@ -1609,7 +1609,7 @@ int iuse::mutagen(player *p, item *it, bool, point)
 {
     if (p->has_trait("MUTAGEN_AVOID")) {
          //~"Uh-uh" is a sound used for "nope", "no", etc.
-        p->add_msg_if_player(m_warning, _("After what happened that last time? uh-uh.  You're not drinking that chemical stuff."));
+        p->add_msg_if_player(m_warning, _("After what happened that last time?  Uh-uh.  You're not drinking that chemical stuff."));
```

Comment:
The non-caps were intentional in these lines: the uh-uh follows immediately after the question, so fast that it's not a new sentence.